bson/src/main/org/bson/json/UuidStringValidator.java

```diff
@@ -22,7 +22,7 @@
     private static final BitSet HEX_CHARS;
 
     static {
-        HEX_CHARS = new BitSet(256);
+        HEX_CHARS = new BitSet('f' + 1);
```

Comment:
[Ignore me]
We could make it even smaller by shifting all char values by `(int)'0'` when converting them to `BitSet` indices.

Reply:
I will ignore you.  I little too clever just to save 64 bits.  ;)